test/unit/object/test_item.py

```diff
@@ -69,10 +91,27 @@ def test_get_shared_link(
     test_item, _ = test_item_and_response
     expected_url = test_item.get_url()
     mock_box_session.put.return_value.json.return_value = {'shared_link': {'url': test_url}}
-    url = test_item.get_shared_link(access, etag=etag)
+    expected_data = {'shared_link': {}}
+    if shared_link_access is not None:
+        expected_data['shared_link']['access'] = shared_link_access
```

Comment:
How about using `@pytest.mark.parametrize` (in a `@genty`-ish way), so each expected_data can be paired with the input?